### PR TITLE
Feature/image cropper

### PR DESCRIPTION
**Pull request by Flight**
Previous PR ( https://github.com/divio/django-filer/pull/656 ) was closed because of github issues.

Image cropper connected, focal point removed.

**Will need BE help here. Can't merge into develop, because it will fully break the detailed page.**

### CODE REVIEW

**Comment by vxsx** [comment]
looks good

File: filer/tests/frontend/unit/test.cropper.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+        fixture.cleanup();
+
+        container = null;
+        image = null;
+        location = null;
+    });
+
+    it('fake test', function () {
+        // TODO
```
Comment (vxsx) [inline filer/tests/frontend/unit/test.cropper.js]: not that important, but when? :)

Comment (Flight) [inline filer/tests/frontend/unit/test.cropper.js]: Monday, January 4, 2016 =)

**Comment by vxsx** [comment]
what's the plan with this?

**Comment by vxsx** [comment]
any progress here?

**Comment by stefanfoulis** [comment]
@Flight @vxsx please ping me when you continue with this. I already have some concept about how to do custom cropping. And I'd like to learn what your approach is about how crop and focal point work together.

**Pull request closed without merging**